Tests/Resources/app/config/config.php

```diff
@@ -12,5 +12,9 @@
 $container->setParameter('cmf_testing.bundle_fqn', 'Symfony\Cmf\Bundle\MediaBundle');
 $loader->import(CMF_TEST_CONFIG_DIR.'/default.php');
 $loader->import(CMF_TEST_CONFIG_DIR.'/phpcr_odm.php');
-$loader->import(__DIR__.'/cmf_media.yml');
+if (version_compare(strtolower(\Symfony\Component\HttpKernel\Kernel::VERSION), '3.0', '<')) {
+    $loader->import(__DIR__.'/cmf_media.yml');
+} else {
+    $loader->import(__DIR__.'/cmf_media_3_0.yml');
```

Comment:
would it maybe be possible to extract the things that are specific to symfony 2 resp 3 into a separate file each that then imports the base file with the rest of configuration? something like cmf_media_2.yml:
```yaml
imports:
    - { resource: cmf_media.yml }

twig:
    form:
        resources:
            - 'CmfMediaBundle:Form:fields.html.twig'
```